### PR TITLE
update xihe-server, async-server, message-server config for points

### PR DESCRIPTION
**Pull request by VictorZhou123**

### CODE REVIEW

**Comment by pkking** [comment]
lgtm

File: applications/xihe-test-v2/async-server/configmap.yaml [hunk 1]
```diff
@@ -72,28 +72,30 @@ data:
...
+        inference_async_start:
+          name: "inference_async_start"
+          topic: "xihe_inference_async_start_test"
+        inference_async_finish:
+          name: "inference_async_finish"
+          topic: "xihe_inference_async_finish_test"
+        picture_publicized:
+          name: "picture_publicized"
+          topic: "xihe_picture_publicized_test"
+        bigmodel_access_log:
```
Comment (zengchen1024) [inline applications/xihe-test-v2/async-server/configmap.yaml:91]: 少了 picture_like

File: applications/xihe-test-v2/message-server/configmap.yaml [hunk 2]
```diff
@@ -147,10 +147,10 @@ data:
     mq:
       address: "${KAFKA_ADDRESS}"
       version: "2.3.0"
-
```
Comment (zengchen1024) [inline applications/xihe-test-v2/message-server/configmap.yaml:150]: 保留空行

File: applications/xihe-test-v2/xihe-server/configmap.yaml [hunk 3]
```diff
@@ -330,7 +353,7 @@ data:
       table:
         pod: pod
         async_task: async_task
-
+
```
Comment (zengchen1024) [inline applications/xihe-test-v2/xihe-server/configmap.yaml:356]: 不要加空格

File: applications/xihe-test-v2/xihe-server/configmap.yaml [hunk 4]
```diff
@@ -159,6 +158,26 @@ data:
...
+        inference_error:
+          name: "inference_error"
+          topic: "xihe_inference_error_test"
+        inference_async_start:
+          name: "inference_async_start"
+          topic: "xihe_inference_async_start_test"
+        inference_async_finish:
+          name: "inference_async_finish"
+          topic: "xihe_inference_async_finish_test"
+        picture_publicized:
```
Comment (zengchen1024) [inline applications/xihe-test-v2/xihe-server/configmap.yaml:174]: 少了 picture_like https://github.com/opensourceways/xihe-server/blob/dev/bigmodel/infrastructure/messageadapter/impl.go#L138

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: applications/xihe-test-v2/async-server/configmap.yaml (hunk 1 above)
Comment (VictorZhou123) [reply applications/xihe-test-v2/async-server/configmap.yaml:94]: > 少了 picture_like

done

File: applications/xihe-test-v2/xihe-server/configmap.yaml (hunk 3 above)
Comment (VictorZhou123) [reply applications/xihe-test-v2/xihe-server/configmap.yaml]: > 不要加空格

done

File: applications/xihe-test-v2/xihe-server/configmap.yaml (hunk 4 above)
Comment (VictorZhou123) [reply applications/xihe-test-v2/xihe-server/configmap.yaml:174]: > 少了 picture_like https://github.com/opensourceways/xihe-server/blob/dev/bigmodel/infrastructure/messageadapter/impl.go#L138

done

File: applications/xihe-test-v2/message-server/configmap.yaml (hunk 2 above)
Comment (VictorZhou123) [reply applications/xihe-test-v2/message-server/configmap.yaml]: > 保留空行

done

File: applications/xihe-test-v2/async-server/configmap.yaml (hunk 1 above)
Comment (zengchen1024) [reply applications/xihe-test-v2/async-server/configmap.yaml:94]: picture_liked key是错误的，要么你修改下xihe的代码

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: applications/xihe-test-v2/async-server/configmap.yaml (hunk 1 above)
Comment (VictorZhou123) [reply applications/xihe-test-v2/async-server/configmap.yaml:94]: > picture_liked key是错误的，要么你修改下xihe的代码

https://github.com/opensourceways/xihe-server/pull/569

**Review by zengchen1024 — APPROVED** [review]

**Pull request merged**